Examples/Detectors/ContextualDetector/src/PayloadDecorator.cpp

```diff
@@ -27,17 +27,16 @@ ActsExamples::ProcessCode ActsExamples::Contextual::PayloadDecorator::decorate(
   std::vector<Acts::Transform3> aStore = m_nominalStore;
 
   ACTS_VERBOSE("New IOV detected, emulate new alignment");
-  if (context.eventNumber % m_cfg.iovSize) {
-    for (auto& tf : aStore) {
-      tf *= Acts::AngleAxis3(m_cfg.rotationStep * context.eventNumber,
-                             Acts::Vector3::UnitY());
-    }
-    // This creates a full payload context, i.e. the nominal store
-    PayloadDetectorElement::ContextType alignableGeoContext;
-    alignableGeoContext.alignmentStore = std::move(aStore);
-    context.geoContext =
-        std::make_any<PayloadDetectorElement::ContextType>(alignableGeoContext);
+  for (auto& tf : aStore) {
+    tf *= Acts::AngleAxis3(
```

Comment:
Ok, makes sense - so there are `m_cfg.iovSize` identical context objects created, and every new patch is caluclated from this integer division.